Client/Ecosia/Migration/EcosiaImport.swift

```diff
@@ -22,6 +26,11 @@ final class EcosiaImport {
 
     struct Failure: Error {
         let reasons: [MaybeErrorType]
+
+        var description: String {
+            // max 3 errors to be reported to save bandwidth and storage
```

Comment:
Just an idea, but if bandwidth and storage are an issue (not sure they are), we could avoid sending error messages and only sending codes, as the messages might be the ones stressing our resources more?

Reply:
I am actually interested in the messages to know why exactly migrations are failing (.e.g. no disk space, corrupt files etc). The codes only tell which part (tabs, favs, history) fails. As those error messages can be long I decided together with Arnaud to cap them to 3 and also remove them from the snowplow database regularly. Also I expect not many errors but you can never know 🤞

Reply:
My idea was mostly having codes for tabs, favs, history, and also codes for the reasons: no disk space = 1, etc., makes them easier to aggregate also. But I'm fine with having messages too.

Reply:
once we know through golden master rollout which errors can occur we can refine the tracking to add more specific codes - for now I don't which errors can actually happen - let's see